accepted/calc-functions.md

```diff
@@ -511,93 +508,152 @@ This algorithm takes a calculation `calc` and returns a number or a calculation.
 
 ## Semantics
 
-### Calculation Expressions
+### `FunctionCall`
 
-Replace the semantics for `CalcExpression`, `ClampExpression`, and `CssMinMax`
-with the following:
+Add the following to [the semantics for `FunctionCall`] before checking for a
+global function:
 
-To evaluate a `UnaryCalcExpression`, `BinaryCalcExpression`, `HypotExpression`,
-`CssMinMax`, `CssRound`, or `CssAbs`:
+[the semantics for `FunctionCall`]: ../spec/functions.md#functioncall
 
-* Let `name` be the lower-case value of the expression's first token without the
-  trailing parenthesis.
+* If `function` is null; `name` is case-insensitively equal to `"min"`, `"max"`,
+  `"round"`, or `"abs"`; and all arguments in `call`'s `ArgumentInvocation` are
+  [calculation-safe], return the result of evaluating `call` [as a calculation].
```

Comment:
This `[as a calculation]` link is also broken. I also looked around and the `[compatible]` link is broken also, but that seems to have been an issue from a previous PR